frontend/src/components/__tests__/DatePicker.js

```diff
@@ -0,0 +1,59 @@
+import '@testing-library/jest-dom';
+import React from 'react';
+import {
+  render, screen, fireEvent, waitFor,
+} from '@testing-library/react';
+
+import { useForm } from 'react-hook-form';
+import DatePicker from '../DatePicker';
+
+describe('DatePicker', () => {
+  // eslint-disable-next-line react/prop-types
+  const RenderDatePicker = ({ minDate, maxDate, disabled }) => {
+    const { control } = useForm();
+    return (
+      <form>
+        <DatePicker
+          control={control}
+          label="label"
+          name="picker"
+          minDate={minDate}
+          maxDate={maxDate}
+          disabled={disabled}
+        />
+      </form>
+    );
+  };
+
+  it('disabled flag works', () => {
```

Comment:
Can you say a little more about what you're testing here?

Reply:
I'll update the test name